src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/CompilerServices/Operators.vb

```diff
@@ -887,13 +887,7 @@ Namespace Microsoft.VisualBasic.CompilerServices
 
         Public Shared Function PlusObject(ByVal Operand As Object) As Object
 
-            If Operand Is Nothing Then
```

Comment:
Dead code, as `GetTypeCode` is implemented as

```vb
Private Shared Function GetTypeCode(ByVal o As Object) As TypeCode
    If o Is Nothing Then
        Return TypeCode.Empty
    Else
        Return o.GetType.GetTypeCode
    End If
End Function
```

and we return `Boxed_ZeroInteger` in the `TypeCode.Empty` case directly below this